addons/forum-post-countdown/post-create.js

```diff
@@ -0,0 +1,10 @@
+export default async function () {
+  const submitButtons = document.querySelectorAll("#djangobbwrap .form-submit [type=submit]");
+  const submitButton = submitButtons[submitButtons.length - 1];
```

Comment:
Can a single page have more than one submit button? I'm curious because I think other addons don't consider this

Reply:
No, they can't - this is from when this addon duplicated the post button, and I wanted to make sure nothing weird could happen.

```suggestion
  const submitButton = document.querySelector("#djangobbwrap .form-submit [type=submit]");
```